gatsby/src/components/Prep/Prep.module.scss

```diff
@@ -0,0 +1,60 @@
+@use '@nice-digital/nds-core/scss/core';
+
+.prepHeading {
+  @include core.nds-h4;
+
+  @include core.mq($from: md) {
+    display: flex;
+    flex-direction: row-reverse;
+  }
+}
+
+.headingIcons {
+  display: block;
+
+  @include core.mq($from: md) {
+    flex: 0 1 20%;
+    text-align: right;
+  }
+}
+
+.headingText {
+  display: block;
+
+  @include core.mq($from: md) {
+    flex: 1 1 80%;
+  }
+}
+
+.manufacturer {
+  font-style: italic;
+}
+
+.packList {
+  list-style: none;
+  margin: 0;
+}
+
+.packItem + .packItem {
+  border-top: 1px solid core.$nds-colour-border;
+}
+
+.packDefinitionList {
+  dt {
+    font-weight: bold;
+
+    @include core.mq($from: md) {
+      display: inline-block;
```

Comment:
Might be easier to wrap each `dt`/`dd` combo in a `div` which we can then style as a `flex` which probably makes the layout easier than using `inline-block`

Reply:
Didn't think this was valid HTML, but turns out it is! https://html.spec.whatwg.org/multipage/grouping-content.html#the-dl-element